jwql/website/apps/jwql/data_containers.py

```diff
@@ -666,6 +666,30 @@ def get_anomaly_form(request, inst, file_root):
     return form
 
 
+def get_group_anomalies(file_root):
+    """Generate form data for context
+
+    Parameters
+    ----------
+    file_root : str
+        FITS filename of selected image in filesystem. May be a
+        file or group root name.
+
+    Returns
+    -------
+    group_anomaly_dict dict
+        root file name key with string of anomalies
+    """
+    # Check for group root name
+    rootfileinfo_set = RootFileInfo.objects.filter(root_name__startswith=file_root).order_by("root_name")
```

Comment:
Will there ever be a case where file_root doesn't return anything? Is the loop below enough to protect against cases like that? e.g. rootfileinfo_set will never be something like [None]?

Reply:
that can theoretically happen, and if it does we will return an empty dict.